cmd/kubeadm/app/cmd/upgrade/common.go

```diff
@@ -54,6 +54,12 @@ type upgradeVariables struct {
 
 // enforceRequirements verifies that it's okay to upgrade and then returns the variables needed for the rest of the procedure
 func enforceRequirements(flags *cmdUpgradeFlags, dryRun bool, newK8sVersion string) (*upgradeVariables, error) {
+
+	// Set the default for the kubeconfig path if the user didn't override with the flags
+	if flags.kubeConfigPath == "" {
+		flags.kubeConfigPath = "/etc/kubernetes/admin.conf"
```

Comment:
should be part of consts.

Reply:
Yup, I want to refactor the flag registration in the upgrade command. _might_ do that as a bugfix in v1.11 still, but not sure yet